findutils/correct-glibc-fflush.patch

```diff
@@ -0,0 +1,143 @@
+From 80cdfba079627e15129a926a133825b961d41e36 Mon Sep 17 00:00:00 2001
+From: Paul Eggert <eggert@cs.ucla.edu>
```

Comment:
I'm not sure if it's better to anonymise these patch files, or to keep them to keep them matching the source exactly. I'd lean towards the latter.

Reply:
This is the format these patch files are posted as to the softwares mailing lists. We should keep them as close to the original as possible for proper attribution and to make it easy to find the context around how these patches were developed and the issue they are intended to solve.